projects/storefrontlib/src/cms-components/configurator/commons/config-group-menu/config-group-menu.component.html

```diff
@@ -0,0 +1,23 @@
+<!--
+<div
+  *ngIf="node?.children.length > 1"
+  class="back is-open"
+  (click)="back()"
+>
+  <h5>
+    <cx-icon [type]="iconType.CARET_LEFT"></cx-icon>
+    {{ 'common.back' | cxTranslate }}
+  </h5>
+</div>
+-->
```

Comment:
Will we need this for our multi-level handling? Otherwise please remove commented code....

Reply:
Yes this would be needed for our multi level handling, I will remove for now.